src/pages/Chat/ChatBubbleSender.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components"
+
+interface Props {
+    value : string,
+    time: string,
+    profileImg: string,
+}
+
+function ChatBubbleSender({value}: Props) {
+  return (
+    <ChatBubbleWrapper>
+        <BubbleText>{value}</BubbleText>
+    </ChatBubbleWrapper>
+  )
+}
+const ChatBubbleWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    width: 297.76px;
```

Comment:
현재 채팅에 width가 고정되어 있는 것 같은데, 이렇게 되면 메시지 길이가 변하더라도 항상 같은 width를 가지는 채팅만 보내지는 것 같아요! max-width로 바꿔주시는게 좋을 것 같습니다!
```suggestion
    max-width: 297.76px;
```
<img width="367" alt="스크린샷 2024-03-31 오전 5 00 34" src="https://github.com/CEOS-Developers/react-messenger-19th/assets/117785708/06baa084-3a19-4e7f-9154-5d2fc8a3a185">